eras/conway/impl/src/Cardano/Ledger/Conway/TxBody.hs

```diff
@@ -576,7 +577,7 @@ pattern ConwayTxBody
         , ctbrOutputs = ctbOutputs
         , ctbrCollateralReturn = ctbCollateralReturn
         , ctbrTotalCollateral = ctbTotalCollateral
-        , ctbrCerts = ctbCerts
+        , ctbrCerts = (SOS.toStrictSeq -> ctbCerts)
```

Comment:
wouldn't a simple function application work here?

Reply:
Ah, ok so this is a pattern match, not a constructor, got it

Reply:
No, it wouldn't. Notice that this is actually a pattern matching here on `ConwayTxBodyRaw`, so the only other way to handle this is to bind it to something else  like `stbCertsOSet` and then add another binding in the `where` clause:
```haskell
  where ctbCerts = SOS.toStrictSeq stbCertsOSet
```
However, since we are already using `ViewPatterns`, I agree with Aniket, that this is a much cleaner approach